src/bundle/Resources/public/js/scripts/admin.anchor.sections.navigation.js

```diff
@@ -0,0 +1,156 @@
+(function (global, doc) {
+    if (!doc.querySelector('.ibexa-navigation-menu')) {
+        return;
+    }
+
+    const SECTION_ADJUST_MARGIN_TOP = 20;
+    const formContainerNode = doc.querySelector('.ibexa-edit-content');
+    const getSecondarySectionActiveItems = () => {
+        const secondarySectionItems = formContainerNode.querySelectorAll(
+            '.ibexa-edit-content__primary-section--active .ibexa-edit-content__secondary-section',
+        );
+
+        return [...secondarySectionItems];
+    };
+    let currentlyVisibleSections = getSecondarySectionActiveItems();
+    const fitSecondarySections = () => {
+        const primarySection = doc.querySelector('.ibexa-edit-content__primary-section--active');
+        const contentColumn = doc.querySelector('.ibexa-main-container__content-column');
+        const firstSection = primarySection.querySelector('.ibexa-edit-content__secondary-section:first-child');
+        const lastSection = primarySection.querySelector('.ibexa-edit-content__secondary-section:last-child');
+        const contentContainer = contentColumn.querySelector('.ibexa-edit-content__container');
+
+        contentContainer.style.paddingBottom = '0px';
+
+        if (!firstSection.isSameNode(lastSection) && lastSection && lastSection.offsetHeight) {
```

Comment:
I think that `lastSection` will never be null, given `firstSection` is not null.
```suggestion
        if (!firstSection.isSameNode(lastSection) && lastSection.offsetHeight) {
```